packages/core/src/anchor/ethereum/ethereum-anchor-service.ts

```diff
@@ -142,12 +123,8 @@ export class EthereumAnchorService implements AnchorService {
   }
 
   async init(): Promise<void> {
-    // Get the chainIds supported by our anchor service
-    const response = await this.#sendRequest(this.#chainIdApiEndpoint)
-    if (response.supportedChains.length > 1) {
-      throw new MultipleChainsError()
-    }
-    this.#chainId = response.supportedChains[0]
+    const supportedChains = await this.#cas.supportedChains()
```

Comment:
Should we handle no chains being returned here, or assume chains will always be returned?

Reply:
There was previously an assumption, that the chains are always returned. Decided not to fix it to focus on the bigger thing. TBH, it is better handled by explicitly parsing the response, and trigger an error if the response is malformed.  Created a ticket to fix it later.